Add SWEEP - Sweet Weapon Enhancement and Expansion Package

Yinsolaya opened the pull request:
* Add Location
https://www.nexusmods.com/newvegas/mods/81043/

* Add tags
- Makes several changes to weapon related records
- Makes changes to text and descriptions
- Makes changes to NPC inventories
- Changes leveled lists

* Add cleaning data
- Version 1.29

Yinsolaya committed:
Add More Modest Type 4 Body and Armor

* Add Bash Tags
- Graphics due to changes to biped data.

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Merge branch 'loot:v0.21' into v0.21

Yinsolaya committed:
Add Vanilla SWEEP

* Add Location
https://www.nexusmods.com/newvegas/mods/81043/

* Add tags
- Makes several changes to weapon related records
- Makes changes to text and descriptions
- Makes changes to NPC inventories
- Changes leveled lists

* Add cleaning data
- Version 1.28

sibir-ine commented on masterlist.yaml:
```diff
@@ -2579,6 +2579,31 @@ plugins:
+      - Stats
+      - Text
+      - WeaponMods
+    dirty:
+      - <<: *quickClean
+        crc: 0xE3A4071D
+        util: '[FNVEdit v4.1.5f](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 52
+    clean:
+      - crc: 0x74E8E658
+        util: 'FNVEdit v4.1.5f'
```
V1.29 was released a few hours ago with a different checksum. Also, we don't include clean checksums for dirty mods since they can easily change:
```suggestion
    dirty:
      - <<: *quickClean
        crc: 0xFF693F51
        util: '[FNVEdit v4.1.5f](https://www.nexusmods.com/newvegas/mods/34703)'
        itm: 52
```

Yinsolaya replied:
Ah! Yeah, didn't expect Sweet to update this one so much recently. I'll make the change on the second one.

Yinsolaya replied:
Probably tomorrow. Wait, what do you mean by alphabetically? Sorry, I'm still learning.

Yinsolaya replied:
Wait, I think I know what you mean. Sorry. Haha.

sibir-ine replied:
Move it after `Vanilla Enhancements.esm`.

sibir-ine reviewed (changes requested):
Looks good, just needs to be sorted alphabetically within the section.

Yinsolaya committed:
Add SWEEP - Sweet Weapon Enhancement and Expansion

Add Location
https://www.nexusmods.com/newvegas/mods/81043/

Add tags

Makes several changes to weapon related records
Makes changes to text and descriptions
Makes changes to NPC inventories
Changes leveled lists
Add cleaning data
Version 1.29

sibir-ine reviewed (approved):
Thanks.
